help/tquery.md

```diff
@@ -0,0 +1,52 @@
+# Tquery
+
+### Typy danych i operatory filtrów
+
+| typ          | null(5) | = | in   | cmp | like | select | sort |
+|--------------|---------|---|------|-----|------|--------|------|
+| bool         | *       | * | .    | .   | .    | *      | +    |
+| date         | *       | * | *    | *   | .    | *      | +    |
+| datetime     | *       | . | .    | *   | .    | *      | +    |
+| int          | *       | * | *    | *   | *    | *      | +    |
+| string       | *       | * | *    | *   | *    | *      | +    |
+| uuid         | *       | * | *    | .   | .    | *      | .    |
+| text         | *       | . | .    | .   | *    | *(8)   | .    |
+| --------(9)  |         |   |      |     |      |        |      |
+| dict(1)      | *       | * | *    | .   | .    | *      | *    |
+| object(2)    | *       | . | .    | .   | .    | *      | .    |
+| dict_list(3) | *       | * | *(7) | .   | .    | *      | .    |
+| list(4)      | *       | . | .    | .   | .    | *      | .    |
+
+- `null` - operator poprawny tylko dla typów nullable
+- `=` - porównanie z wartością tego samego typu, int-int, string-string, itp., brak możliwości porównania z null ani z
+  pustym stringiem
+- `==`(?) - porównanie typu binary, bardziej kosztowne dla sql'a, ale z zachowaniem wielkości liter, polskich liter,
+  białych znaków
+- `in` - zawiera się w niepustej liście, reguły jak dla `=`
+- `cmp` - `<`, `<=`, `>`, `>=`, reguły typów jak dla `=`
+- `like` - wartość to niepusty string
+  - `%v`, `v%`, `%v%` - w wartości `%` i `_` są escape'owane
+  - możliwy `v`, gdzie wartość nie jest escape'owana
```

Comment:
Taki ewentualny operator to by się raczej nazywał like. Bo teraz w v% to ten % oznacza że to jest like, a w samym v to nie będzie tego widać.

Reply:
może być